.github/workflows/delegator_generic_slsa3.yml

```diff
@@ -252,11 +252,8 @@ jobs:
           SLSA_VERSION: ${{ fromJson(needs.verify-token.outputs.slsa-verified-token).slsaVersion }}
         run: |
           case "$SLSA_VERSION" in
-            "v0.2")
-              echo "predicate-type=https://slsa.dev/provenance/v0.2" >> "$GITHUB_OUTPUT"
-              ;;
-            "v1-rc1")
-              echo "predicate-type=https://slsa.dev/provenance/v1-rc1" >> "$GITHUB_OUTPUT"
+            "v0.2" | "v1.0")
+              echo "predicate-type=https://slsa.dev/provenance/$SLSA_VERSION" >> "$GITHUB_OUTPUT"
```

Comment:
In https://slsa.dev/provenance/v1 the provenance predicateType is just `v1`, do you mean to have v1.0?

Reply:
hp, thanks. Will update, thank you!